src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
+import { Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import groupService from '@services/groups.service';
+
+class GroupController {
+  public groupService = new groupService();
+
+  public getGroups = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const findAllGroupsData: Group[] = await this.groupService.findAllGroup();
```

Comment:
What if they're thousands of groups, please add a `limit` with pagination
```suggestion
      const findAllGroupsData: Group[] = this.groupService.findAllGroup({page: 1, limit: 10, sortBy: 'name', sortOrder: 'asc'});

```